services/brig/src/Brig/API/Internal.hs

```diff
@@ -366,13 +366,12 @@ deleteFederationRemote dom = do
     queue = routingKey defederationQueue
 
 -- | Remove one-on-one conversations for the given remote domain. This is called from Galley as
--- part of the defederation process, and should not be called duriung the initial domain removal
+-- part of the defederation process, and should not be called during the initial domain removal
 -- call to brig. This is so we can ensure that domains are correctly cleaned up if a service
 -- falls over for whatever reason.
 deleteFederationRemoteGalley :: Domain -> ExceptT Brig.API.Error.Error (AppT r) ()
 deleteFederationRemoteGalley dom = do
   lift . wrapClient . Data.deleteRemoteConnectionsDomain $ dom
-  assertNoDomainsFromConfigFiles dom
```

Comment:
the reason for this line is that if you delete a remote that's also present in the config file, it will fail silently (signal success), as read is always the union of cassandra and yaml, and yaml can't be changed here.  there is a comment about this somewhere in this CRUD family.

why did you remove it?